api/php/DBgateways/RestaurantDbGateway.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+include_once "php/config.php";
+
+class RestaurantDbGateway {
+
+    // private $servername = "localhost";
+    // private $username = "root";
+    // private $password = "foody";
+    // private $db = "foody";
+
+
+    private $dbConnection = null;
+
+    public function __construct() {
+        $this->dbConnection = new DbConnecter();
+    }
+
+    // public function getRestaurants () {
+
+    // }
+
+    public function validateRestaurant (string $restaurantName, string $restaurantAddress, string $restaurantPhone, string $restaurantWebsite) {
```

Comment:
I think we can change the name of this function. RestaurantDBGateway class should follow single responsibility principle.  The class methods should only connect and query data from the database. All validation needs to be done in the service. So we can break this function down into two simple functions, a validation in the RestaurantService and a getRestaurant at the DbGateway.